src/patternfly/components/Page/page-header-tools-item.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-page__header-tools-item{{#if page-header-tools-item--modifier}} {{page-header-tools-item--modifier}}{{/if}}"
```

Comment:
I don't see this being used, do we need to keep it?

Reply:
nope!